.github/PULL_REQUEST_TEMPLATE/integration.md

```diff
@@ -0,0 +1,28 @@
+# Integration
```

Comment:
A lot of stuff here are things that only we can update, like corp docs (I mean, an outside contributor could make a PR for our docs, but maybe we should own that)